backend/src/main/java/ch/puzzle/okr/controller/KeyResultController.java

```diff
@@ -12,15 +13,21 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
+
 
 @RestController
 @RequestMapping("api/v1/keyresults")
 public class KeyResultController {
     KeyResultService keyResultService;
 
+
+    KeyResultRepository keyResultRepository;
     KeyResultMapper keyResultMapper;
```

Comment:
Alle private final machen wie in allen anderen Controllern
Weniger Lücken dazwischen also Zeile 23 und 24. Sieht schöner aus.